draft-ietf-quic-transport.md

```diff
@@ -910,7 +910,7 @@ diagnostic or security purposes.
 For servers, packets that aren't associated with a connection potentially create
 a new connection.  However, only packets that use the long packet header and
 that are at least the minimum size defined for the protocol version can be
-initial packets.  Unless the server is buffering 0-RTT packets, a server MUST
+initial packets.  Unless the server is buffering 0-RTT packets, a server MAY
```

Comment:
This is an odd sentence now. Remove the phrase "Unless the server is buffering 0-RTT packets"